pdns/dnsdistdist/docs/reference/custommetrics.rst

```diff
@@ -10,13 +10,19 @@ Then you can update those at runtime using the following functions, depending on
  * manipulate counters using :func:`incMetric` and  :func:`decMetric`
  * update a gauge using :func:`setMetric`
 
-.. function:: declareMetric(name, type, description [, prometheusName]) -> bool
+.. function:: declareMetric(name, type, description [, prometheusName|options]) -> bool
 
   .. versionadded:: 1.8.0
 
   .. versionchanged:: 1.8.1
     This function can now be used at runtime, instead of only at configuration time.
 
+  .. versionchanged:: ?
```

Comment:
For now let's go with 2.0.0. I'll update the docs if this gets backported to 1.9.x.